symfony.lock

```diff
@@ -69,7 +69,7 @@
         ]
     },
     "php": {
-        "version": "7.3"
+        "version": "7.4"
```

Comment:
not sure about this change

Reply:
Automatically made when I ran `composer remove doctrine/inflector`.

Lock files are not committed in other project.
Since it depends on the environment, I'm not sure what to do...

Reply:
On bundles it should not be commited, but on projects that get deployed it is normal. IMO you should revert this change